### PR TITLE
First run through the code to cleanup styling

### PR DESCRIPTION
**Pull request by elehmer**
This is a BIG PR that cleans up the code for styling and formatting. It is a first pass and will need to be reviewed carefully to see that I haven't introduced any syntax errors.

@bkg Please see if I've screwed up the PEP8 styling and make edits where appropriate. Thanks!

### CODE REVIEW

**Comment by elehmer** [comment]
Well, there is a syntax error already lol This will take some time to sort out...

**Comment by nicolejkeeney** [comment]
> Hmm, I branched off of `catalog_restructure` and I was hoping it would merge to that branch but it wants to merge to `main`

Looks fine to me. It contains the code from catalog_restructure. We just need to wait until that branch is merged to merge this one

**Review by nicolejkeeney — APPROVED** [review]
This looks good to me. Obviously just need to wait to merge until the catalog_restructure branch is merged.

**Review by nicolejkeeney — APPROVED** [review]
This looks good to me. Obviously just need to wait to merge until the catalog_restructure branch is merged.

File: climakitae/core.py
Comment (vicford) [inline climakitae/core.py:15]: Additional line in between imports and class definition?

File: climakitae/core.py
Comment (vicford) [inline climakitae/core.py:50]: Select and retrieve both have two lines of whitespace, but only one ahead of view?

File: climakitae/core.py
Comment (vicford) [inline climakitae/core.py:69]: Same here, only one line of whitespace (vs two above)

**Comment by vicford** [comment]
I'm noticing some scripts have a line of whitespace between a function and the resulting return call, and some do not

File: climakitae/data_loaders.py
Comment (vicford) [inline climakitae/data_loaders.py:16]: two lines here?

File: climakitae/data_loaders.py
Comment (vicford) [inline climakitae/data_loaders.py:48]: but one line here

File: climakitae/data_loaders.py
Comment (vicford) [inline climakitae/data_loaders.py:234]: and two here

File: climakitae/metadata_update.py
Comment (vicford) [inline climakitae/metadata_update.py:9]: line of whitespace here?

File: climakitae/selectors.py
Comment (vicford) [inline climakitae/selectors.py:132]: No line of whitespace before return statement above when others do?

File: climakitae/selectors.py
Comment (vicford) [inline climakitae/selectors.py:329]: 3 lines of whitespace, when we've used 2 elsewhere

File: climakitae/selectors.py
Comment (vicford) [inline climakitae/selectors.py:13]: Other scripts have a single line between our import statements and our from import statements

**Comment by vicford** [comment]
threshold_tools.py has `# =========` breaks in between each function, the rest of our scripts do not do this.

File: climakitae/threshold_tools.py
Comment (vicford) [inline climakitae/threshold_tools.py:408]: from conf_int and down, the functions that have many items in the definition call have their ending `):` in line with the def call, is this right?

**Review by vicford — COMMENTED** [review]
This looks great Eric. I left some comments about what I mainly noticed in differences in whitespace lines, but if this is not a big concern for us, then almost all my comments can be ignored

**Comment by elehmer** [comment]
> I'm noticing some scripts have a line of whitespace between a function and the resulting return call, and some do not
> 
> with whitespace: catalog_utils derive_variables metadata_update selectors (some) threshold_tools timeseriestools (some) tmy (some) unit_conversions (some) utils (some) warming_levels (some)

Yeah, I waffled on this one. Technically, the `return` should be right after the code with no extra white space. I will impose that.

File: climakitae/core.py
Comment (elehmer) [reply climakitae/core.py:48]: Yep! Thanks for the catch. Fixed

File: climakitae/explore.py
Comment (elehmer) [reply climakitae/explore.py]: Ok, finally figured out how to properly format this lol

**Comment by elehmer** [comment]
Second pass through the repo complete. Amazing how much you miss even when you are concentrating. I finally figured out how to split large strings using `( " " " " " ")` so text strings should display properly. I will now do some serious testing of all the code.

**Comment by nicolejkeeney** [comment]
Hey Eric, so sorry about the bit of a mess I created when trying to resolve merge conflicts in this branch. You'll see that I had to do a force-commit back to your most recent commit to undo what I did. I created a mess, but it's all cleaned up now and back to where you left it. 

I also changed the base branch to catalog_restructure :D 

**Regarding the merge conflicts: ** Victoria just merged some code in `tmy.py` that has conflicts. There is also a merge conflict in the `data_explore.py` file, perhaps due to the merge of PR https://github.com/cal-adapt/climakitae/pull/105 on Oct 20? I created a mess trying to resolve this, so I will leave that to you to resolve and stop messing with your code!

**Review by nicolejkeeney — CHANGES_REQUESTED** [review]
Just resolve the merge conflicts and it is good to me :D

**Pull request merged**
